FirebaseInAppMessaging/Swift/Source/SwiftUIPreviewHelpers.swift

```diff
@@ -14,23 +14,24 @@
 
 import FirebaseInAppMessaging
 
+@available(iOS 13.0, macOS 10.15, macCatalyst 13.0, tvOS 13.0, *)
 public struct InAppMessagingPreviewHelpers {
```

Comment:
If we're gating `InAppMessagingPreviewHelpers` with this availability check (that includes `tvOS 13.0`, are the added background color APIs necessary?